adhocracy-plus/assets/scss/components/_label.scss

```diff
@@ -4,7 +4,8 @@
     color: contrast-color($text-color);
     font-size: $font-size-xs;
     font-weight: normal;
-    border-radius: 0.4em;
+    border-radius: 0.8em;
+    border: 1px solid $text-color;
     padding: 0.2em 0.7em;
 }
 
```

Comment:
again we use `label` on many elements so updating needs to be something thats considered carefully, checking the component library can help when changing these things to see if it has unintended effects, this will also help you see the danger colour change issues. I would just apply what we need to the `live--label` class. Border radius should be something thats the same for all labels so we can change it here or check with designer, I normally go with whichever is easier or I think looks better :p